test/timeout.js

```diff
@@ -10,26 +10,44 @@ test.before('setup', async () => {
 	s = await createServer();
 
 	s.on('/', async (req, res) => {
-		await delay(20);
+		await delay(80);
 		res.statusCode = 200;
 		res.end('OK');
 	});
 
 	await s.listen(s.port);
 });
 
-test('timeout option', async t => {
-	const err = await t.throws(got(`${s.url}/`, {
-		timeout: 1,
+test('timeout option (ETIMEDOUT)', async t => {
+	const err = await t.throws(got(`${s.url}`, {
```

Comment:
No point in using a template literal when there's no other content than the variable.